platform/security/policy/security-policy-context/src/test/java/org/codice/security/policy/context/PolicyManagerTest.java

```diff
@@ -108,6 +109,10 @@ public void setup() {
                 new Policy("/aaa/aaa", null, new ArrayList<String>(), null));
         manager.setContextPolicy("/foo/bar",
                 new Policy("/foo/bar", null, new ArrayList<String>(), null));
+        manager.setContextPolicy(
+                "/foo/bar/baz/blah/bing/bab/bozo/bongo/bingo/burp/blaze/blurb/blunt/block",
```

Comment:
While these are cute, it might be easier to double check if it was `/1/2/3/...` instead.